packages/editor/src/components/rich-text/index.native.js

```diff
@@ -92,35 +98,32 @@ export class RichText extends Component {
 	 * handler.
 	 *
 	 */
-	splitContent( htmlText, start, end ) {
+	splitContent( currentRecord, blocks = [], isPasted = false ) {
 		const { onSplit } = this.props;
 
 		if ( ! onSplit ) {
```

Comment:
Not strictly related to this PR but, it seems that the `onSplit` prop is already marked as deprecated on the web side so, since we're touching the `splitContent` method, what do you think about mirroring this (https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/rich-text/index.js#L84-L93) here too?

I'd suggest a follow up PR with that since it's not a blocker for this PR and adding more diff to this is not desirable :)